lib/rspec/rails/view_controller.rb

```diff
@@ -0,0 +1,43 @@
+module RSpec
+  module Rails
+    # Methods added to ActionView::TestCase::TestController, only in view specs
+    module ViewController
+      def self.included(_klass)
```

Comment:
I hadn't thought about that distinction. The intention is to include them only for view specs. It's an awkward fit because the user has to set them somehow, and they've already got a controller object available. I thought about surfacing a whole different object, but the extra params are not unrelated to the controller, request, etc. 

That said - what's the right way to include those methods in the controller class just for view specs?